core/frontend/src/tile/ImdlReader.ts

```diff
@@ -223,13 +223,12 @@ export class ImdlReader extends GltfReader {
   }
 
   private async readNamedTexture(namedTex: any, name: string): Promise<RenderTexture | undefined> {
-    // If the texture has a valid Id64 name, it is valid to request it directly from the backend.
-    // If the texture does not have a valid Id64 name, it is not valid to request it directly from the backend.
-    // In that case, we instead retrieve the texture directly from the tile.
-    // Furthermore, if external textures are disabled, we retrieve the texture directly from the tile in both cases.
-    // Only do this if bufferViewJson does not exist.
-    // (If external textures are enabled and the backend provides a bufferViewJson, that means the image data
-    // is small enough that the backend decided to embed that data rather than make it an external texture.)
+    // Reasons a texture could be embedded in the tile content instead of requested separately from the backend:
+    // - external textures are disabled
+    // - the texture name is not a valid Id64 string
+    // - the texture is below a certain backend-hardcoded size threshold
+    // The bufferViewJson being undefined signifies any of the above conditions. In that case, the texture contents

```

Comment:
The bufferViewJson being **defined** signifies these conditions; being undefined signifies the image is not embedded.

Reply:
yeah; clearly meant defined... thanks for the catch.  Reworded that comment.